src/apps/tezos/writers.py

```diff
@@ -0,0 +1,24 @@
+from apps.common.writers import (
+    write_bytes,
+    write_uint8,
+    write_uint32_be,
+    write_uint64_be,
+)
+
+write_uint8 = write_uint8
+write_uint32 = write_uint32_be
+write_uint64 = write_uint64_be
+write_bytes = write_bytes
+
+
+def write_bool(w: bytearray, boolean: bool):
```

Comment:
This function is already in `sign_tx.py`. It would be cool to move all writers from `sign_tx.py` to `writers.py`, so we do not have the same function defined multiple times :)

Reply:
yes, please do that :)

Reply:
looking back at this, does it make sense to have a `writers.py` just for a `write_bool` function? maybe moving it e.g. to `helpers.py` would be better?